src/transformers/models/wav2vec2/convert_wav2vec2_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -163,11 +174,37 @@ def convert_wav2vec2_checkpoint(
         config = Wav2Vec2Config()
 
     if is_finetuned:
+        if dict_path:
+            target_dict = Dictionary.load(dict_path)
+
+            config.bos_token_id = target_dict.bos_index
+            config.eos_token_id = target_dict.eos_index
+            config.pad_token_id = target_dict.pad_index
+            config.vocab_size = len(target_dict.symbols)
+            vocab_path = os.path.join(pytorch_dump_folder_path, "vocab.json")
+            with open(vocab_path, "w", encoding="utf-8") as vocab_handle:
+                json.dump(target_dict.indices, vocab_handle)
+            tokenizer = Wav2Vec2CTCTokenizer(
+                vocab_path,
+                unk_token=target_dict.unk_word,
+                pad_token=target_dict.pad_word,
+                bos_token=target_dict.bos_word,
+                eos_token=target_dict.eos_word,
+                word_delimiter_token="|",
+                do_lower_case=False,
+            )
+            feature_extractor = Wav2Vec2FeatureExtractor(
+                feature_size=1, sampling_rate=16000, padding_value=0, do_normalize=True, return_attention_mask=True
```

Comment:
The value for `return_attention_mask` depends on whether a model has been pre-trained using attention mask. For models with `feat_extract_norm='group'` it should be False and for `feat_extract_norm='layer'` True:

```
feature_extractor = Wav2Vec2FeatureExtractor(
                feature_size=1, sampling_rate=16000, padding_value=0, do_normalize=True, return_attention_mask=True if config.feat_extract_norm=='layer' else False
            )
```

Reply:
I didn't know that, fixed it now~ Thank you